python/1manifoldLearning.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+from paraview.simple import *
+
+# create a new 'CSV Reader'
+pointCloudcsv = CSVReader(FileName=['pointCloud.csv'])
+
+# create a new 'TTK DimensionReduction'
+tTKDimensionReduction1 = TTKDimensionReduction(Input=pointCloudcsv, ModulePath='default')
+tTKDimensionReduction1.InputColumns = ['Points:0', 'Points:1', 'Points:2']
+tTKDimensionReduction1.UseAllCores = 0
+
+# create a new 'Table To Points'
+tableToPoints2 = TableToPoints(Input=tTKDimensionReduction1)
+tableToPoints2.XColumn = 'Component_0'
+tableToPoints2.YColumn = 'Component_1'
+tableToPoints2.ZColumn = 'Component_0'
+tableToPoints2.a2DPoints = 1
+
+# create a new 'Gaussian Resampling'
+gaussianResampling2 = GaussianResampling(Input=tableToPoints2)
+gaussianResampling2.ResampleField = ['POINTS', 'ignore arrays']
+gaussianResampling2.ResamplingGrid = [128, 64, 3]
+gaussianResampling2.SplatAccumulationMode = 'Sum'
+
+# create a new 'Table To Points'
+tableToPoints1 = TableToPoints(Input=pointCloudcsv)
+tableToPoints1.XColumn = 'Points:0'
+tableToPoints1.YColumn = 'Points:1'
+tableToPoints1.ZColumn = 'Points:2'
```

Comment:
I don't believe this line is necessary.
Thanks for double checking (e.g. with diff or paraview) that the outputs of the script are unchanged after the edit.